src/main/groovy/org/assertj/generator/gradle/internal/tasks/AssertionsGeneratorReport.groovy

```diff
@@ -44,127 +45,135 @@ class AssertionsGeneratorReport {
+        }.toString()
+    }
+
+    private StringBuilder buildGeneratorReportSuccess(final StringBuilder self) {
+        self.tap {
+            append(Line)
+            append("Directory where custom assertions files have been generated:\n")
+            append(INDENT).append(directoryPathWhereAssertionFilesAreGenerated).append(Line)
+            append(Line)
+            append("Custom assertions files generated:\n")
+            for (String fileName : generatedCustomAssertionFileNames) {
+                append(INDENT).append(fileName).append(Line)
+            }
+            if (!inputClassesNotFound.isEmpty()) {
+                append(Line)
+                append("No custom assertions files generated for the following input classes as they were not found:\n")
+                for (inputClassNotFound in inputClassesNotFound) {
+                    append(INDENT).append(inputClassNotFound).append(Line)
+                }
             }
+            reportEntryPointClassesGeneration(it)
         }
-        reportEntryPointClassesGeneration(reportBuilder)
     }
 
-    private void reportEntryPointClassesGeneration(StringBuilder reportBuilder) {
-        for (AssertionsEntryPointType type : assertionsEntryPointFilesByType.keySet()) {
-            if (assertionsEntryPointFilesByType.get(type) != null) {
-                String entryPointClassName = remove(type.getFileName(), ".java")
-                reportBuilder.append(System.lineSeparator())
-                        .append(entryPointClassName).append(" entry point class has been generated in file:\n")
-                        .append(INDENT).append(assertionsEntryPointFilesByType.get(type).getAbsolutePath())
-                        .append(System.lineSeparator())
+    private StringBuilder reportEntryPointClassesGeneration(final StringBuilder self) {
+        self.tap {
+            for (type in assertionsEntryPointFilesByType.keySet()) {
+                if (assertionsEntryPointFilesByType.get(type) != null) {
+                    String entryPointClassName = remove(type.getFileName(), ".java")
+                    append(Line)
+                            .append(entryPointClassName).append(" entry point class has been generated in file:\n")
+                            .append(INDENT).append(assertionsEntryPointFilesByType.get(type).getAbsolutePath())
+                            .append(Line)
+                }
             }
         }
-    }
 
-    private void buildGeneratorReportWhenNothingWasGenerated(StringBuilder reportBuilder) {
-        reportBuilder.append(System.lineSeparator())
-        reportBuilder.append("No assertions generated as no classes have been found from given classes/packages.\n")
-        if (isNotEmpty(inputClasses as String[])) {
-            reportBuilder.append(INDENT).append("Given classes : ").append(Arrays.toString(inputClasses))
-            reportBuilder.append(System.lineSeparator())
-        }
-        if (isNotEmpty(inputPackages as String[])) {
```

Comment:
These incantations were replaced with `!field?.isEmpty()` which groovy coaleces `null -> false` and is _much_ easier to grep.